### DISCUSSION TITLE
Display Line Breaks?

### DISCUSSION THREAD

**Discussion by l008com** [HexFiend/HexFiend#454] [General]
Hi. Is there a way to have Hex Fiend render line breaks AS line breaks? My guess is no, but this would be a super easy feature for the dev to implement. 

The reason it matters is because sometimes you need to browse extremely large files, and this is the ONLY program I know of that can open/read/view a large file without trying to load it all into RAM. Files that make BBEdit weep, open right up in HexFiend. But if we could see line breaks AS line breaks, then it would be super useful to use more often when we need to take little samples of extremely large files.

**Comment by kainjow** [comment]
No, there’s not a way. See previous issue: https://github.com/HexFiend/HexFiend/issues/57

Personally I use VSCode nowadays for large text files.
